Plots/Regression/NCL_regress_1.py

```diff
@@ -0,0 +1,83 @@
+"""
+NCL_regress_1.py
+===============
+This script illustrates the following concepts:
+   - Read tabular values from an ascii file
+   - Calculating the least squared regression for a one dimensional array
+   - Drawing a scatter plot with a regression line
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/regress_1.ncl
+    - Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/regress_1_lg.png
+"""
+
+###############################################################################
+# Import packages
+import numpy as np
+import matplotlib.pyplot as plt
+
+import geocat.datafiles as gdf
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Read in data:
+
+# Open a ascii data file using numpy's loadtxt
+# Specify the delimiter from the file
+# Read in the data as a floats
+x, y = np.loadtxt(gdf.get("ascii_files/regress_1.txt"),
+                  delimiter=',',
+                  unpack=True,
+                  dtype=float)
+
+###############################################################################
+# Calculate regression:
+
+# Use numpy's polyfit to calculate a linear regression
+# This function outputs [a, b], where y = ax + b in the regression fit
+[a, b] = np.polyfit(x, y, 1)
+
+###############################################################################
+# Transform the found regression model for plotting
+
+# Now that we have a y = ax + b, we can evaluate this model over a range,
+# which will give us points to plot
+x_regress = range(int(min(x)), int(max(x)))
+y_regress = a * x_regress + b
+
+###############################################################################
+# Plot
+
+# Generate figure (set its size (width, height) in inches) and axes
+plt.figure(figsize=(6, 6))
+ax = plt.gca()
+
+# Plot original data
+plt.scatter(x, y, c='r', s=4)
```

Comment:
In the NCL script, the comments say this example shows how to change the color and size of markers. Since the kwargs `c` and `s` aren't very descriptive, could you either a) add a comment to this explaining what those kwargs change or b) spell out `color` and `size` as well as the color name? This will make it easier for novice Python programmers to understand.

If you change this, you may want to add changing marker size/color to the list of demonstrated concepts in the beginning comments.

Reply:
Great point! I expanded c to color, but s cannot be expanded to size. I have replaced c with color, and left an explanation of the meaning of s, as well as added to the list of demonstrated concepts